advanced/Scripts/version.sh

```diff
@@ -89,14 +120,19 @@ EOM
 	exit 0
 }
 
+PHVERSION=$(getLocalVersion "${PHGITDIR}")
```

Comment:
If we do
```
pihole -v -a -c; pihole -v -a -l; pihole -v -p -c; pihole -v -p -l
```
this does not only take a full six seconds of time, but also it queries the Github API 8 times (2x for each run), where it would only be necessary to do this twice!

Why do we process all of them always?